cassandra-config/pom.xml

```diff
@@ -0,0 +1,49 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                      http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.lightbend.lagom.sample.chirper</groupId>
+        <artifactId>activator-lagom-java-chirper</artifactId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>cassandra-config</artifactId>
```

Comment:
I was under the impression that this `cassandra-config` pseudo-project is no longer needed. See https://github.com/lagom/lagom/issues/566